parity-genesis.template.json

```diff
@@ -8,7 +8,7 @@
         "accountStartNonce": "0x0",
         "maximumExtraDataSize": "0x20",
         "minGasLimit": "0x1388",
-        "networkID" : "0x7CE",
+        "networkID" : "0xFD5C9EC",
```

Comment:
Can we add a comment indicating which chain this is?

Reply:
I don't believe so, as it's a JSON file.